torch/_torch_docs.py

```diff
@@ -4365,6 +4365,44 @@ def merge_dicts(*dicts):
             [1.0311, 0.3901, 0.5049]])
 """)
 
+add_docstr(torch.moveaxis,
+           r"""
+moveaxis(input, src, dst) -> Tensor
+
+Move axes of an array to new positions.
```

Comment:
This needs to be clearer about which dimensions are moving where. Also: what does it mean for a dimension to "move?"